lib/yform/value/index.php

```diff
@@ -43,11 +43,11 @@ public function postFormAction(): void
 
             if ($relations) {
                 foreach ($this->getRelationValues($relations) as $v) {
-                    $value .= ' '.$v;
+                    $values[] = $v;
                 }
             }
-
-            $value .= $this->getElement('salt');
+            $value = implode(" ", $values);
+            $value = $this->getElement('salt');
```

Comment:
Hier fehlt ein Punkt "."
Bzw. der Salt müsste auch noch in den Array bevor dieser implodiert.

Oder doch die einfache Variante mit einem trim(). Kann man als PR auch viel leichter erfassen ;-)

Reply:
Das war eben meine Frage. Können auch trim nehmen, ich hatte das Gefühl, das wäre eine Krücke.

Reply:
Ich weiß was du meinst, aber ich habe das Gefühl trim() würde mit 6 zusätzlichen Zeichen, das Problem lösen. Und ein gewisser Jan K. könnte das innerhalb 5 Sekunden erfassen und auf "merge" klicken. ;-)

Reply:
Vielleicht kann @dergel einfach mitteilen, wie er es gerne hätte, dann ist es flugs erledigt.